File: Level-1/34.java
import java.util.*;
class Solution {
    public String[] solution(int n, int[] arr1, int[] arr2) {
        String[] answer = new String[n];
        Arrays.fill(answer, "");
        for(int i = 0; i < n; i++){
            String s1 = Integer.toBinaryString(arr1[i]);
            String s2 = Integer.toBinaryString(arr2[i]);
            s1 = "0".repeat(n - s1.length()) + s1;
            s2 = "0".repeat(n - s2.length()) + s2;
            for(int j = 0; j < n; j++){
                int n1 = s1.charAt(j) - '0';
                int n2 = s2.charAt(j) - '0';
                if(n1 == 0 && n2 == 0){
                    answer[i] += " ";
                }else{
                    answer[i] += "#";
                }
            }
        }
        return answer;
    }
}

File: Level-1/34.js
function solution(n, arr1, arr2) {
    let answer = [];
    
    for(let i = 0; i < n; i++) {
        let a = arr1[i].toString(2);
        let b = arr2[i].toString(2);
        a = '0'.repeat(n - a.length) + a;
        b = '0'.repeat(n - b.length) + b;
        
        let s = '';
        for(let j = 0; j < n; j++) {
            if(a[j] === '1' || b[j] === '1') {
                s += '#';
            }else {
                s += ' ';
            }
        }
        
        answer.push(s);
    }
    
    return answer;
}
